scripts/compatibility_report.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# check if we are inside a pipenv virtualenv
+pipenv --venv
+if test $? -eq 0
+then
+    array=( "0.5" "0.6" "0.6.1" "0.7" )
+    for i in "${array[@]}"
+    do
+        yes | pip uninstall torchvision
```

Comment:
MAYBE: Use ```pip uninstall -y torchvision```

Reply:
Much better! That should work on windows now too. 👍